.github/workflows/prerelease-test.yaml

```diff
@@ -0,0 +1,46 @@
+# Push the release branch to upstream/prerelease_test branch
```

Comment:
Can you please elaborate on the convention with the branch name to run the tests?

Reply:
Added, picked from Release Wiki [here](https://github.com/timescale/eng-database/wiki/Releasing-TimescaleDB#prerelease-testing) , as we want to do away with this Wiki page eventually.